components/k8s-model-server/images/README.md

```diff
@@ -0,0 +1,17 @@
+# Building and testing the TF serving image.
+
+The workflow will checkout the source code from kubeflow/kubeflow, then build and push the image.
+
+Deploy the workflow by:
+```
+ks apply prow -c workflows
+```
+
+## Checkout
+The version of kubeflow is determined by the parameters such as `PULL_NUMBER` or `PULL_BASE_SHA`.
+See [checkout.sh](https://github.com/kubeflow/testing/blob/master/images/checkout.sh) for more detail.
```

Comment:
Add a link to this list here:
https://github.com/kubernetes/test-infra/tree/master/prow#job-environment-variables